pkg/ffmpeg/codec_hardware.go

```diff
@@ -354,7 +389,8 @@ func (f *FFMpeg) hwCodecMaxRes(codec VideoCodec) (int, int) {
 	case VideoCodecN264,
 		VideoCodecN264H,
 		VideoCodecI264,
-		VideoCodecI264C:
+		VideoCodecI264C,
+		VideoCodecRK264:
 		return 4096, 4096
```

Comment:
The Rockchip encoder supports up to 8192:
https://github.com/nyanmisaka/ffmpeg-rockchip/blob/cc5de845d52265dd25cea3738952d331708c2374/libavfilter/vf_vpp_rkrga.c#L200-L204